projects/observability/src/pages/custom-dashboards/custom-dashboard-panel/custom-dashboard-panel.component.ts

```diff
@@ -114,4 +141,8 @@ export class CustomDashboardPanelComponent implements OnInit {
     event.stopPropagation();
     this.deletePanel.emit({ panelName: this.panel!.name, panelId: this.panel!.id });
   }
+  public onPanelClone(event: MouseEvent): void {
+    event.stopPropagation();
```

Comment:
Can you add a comment around why we have had to use `event.stopPropagation` here.